lib/backdrop.dart

```diff
@@ -273,11 +273,14 @@ class _BackdropScaffoldState extends State<BackdropScaffold>
     return Material(
       elevation: 12.0,
       borderRadius: widget.frontLayerBorderRadius,
```

Comment:
Now that we use `ClipRRect` I think we can even remove the `Material()` widget that wraps it. I have tried it locally and apparently it doesn't change anything...
What do you think @danielborges93?

Reply:
According to the Material Design documentation, the front layer has a elevation of 1 (it applies a very small shadow). Maybe, because this, it makes sense keep it with the correct value (1? 16? I don't know hehe). But there are almost insignificant visual changes. I can remove it without problems.

Reply:
Oh, I see. You're right.
Well then I would say we should set the value to 1.0, like the specification says.

Reply:
Elevation value changed to 1.0 😉 I am happy to contribute with the project!

Reply:
Thanks @danielborges93 for the contribution